zip-0316.rst

```diff
@@ -407,10 +408,13 @@ The following UFVK or UIVK Encodings are used in place of the
   treated as unrecognized by Consumers.
 
 * For Transparent P2PKH Addresses that are derived according to BIP 32
-  [#bip-0032]_ and BIP 44 [#bip-0044]_, the UFVK and UIVK Encodings
-  have Typecode :math:`\mathtt{0x00}.` These encodings are identical
-  and are the serialization of an extended public key, defined in the
-  section “Serialization format” of BIP 32 [#bip-0032-serialization-format]_.
+  [#bip-0032]_ and BIP 44 [#bip-0044]_, the FVK and IVK Encodings
+  have Typecode :math:`\mathtt{0x00}.` Both of these encodings are the
+  serialization of an extended public key, defined in the section
+  “Serialization format” of BIP 32 [#bip-0032-serialization-format]_.
```

Comment:
In ZIP sync today, we agreed to alter this to instead just encode:
- The chain code.
- The public key data.

This is effectively the suffix of the extended public key encoding, leaving out the potentially-private data (e.g. child index which is the account). The chain code `c` is necessary to include, as we leverage the capability to derive children as a transparent diversification mechanism.